back/engines/commercial/analysis/app/lib/analysis/LLM/claude_2_1.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Analysis
+  module LLM
+    class Claude21 < AnthropicClaude
+      def context_window
+        # From https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-claude.html:
+        # To avoid timeouts with Anthropic Claude version 2.1, we recommend limiting the input
+        # token count in the prompt field to 180K. We expect to address this timeout issue soon.
+        180_000
+      end
+
+      def accuracy
```

Comment:
@kogre I'm not entirely sure yet what to pick here. So I tended to go for lower values, as models like GPT4 currently seem somewhat better.